driver/driver.go

```diff
@@ -93,37 +116,181 @@ func (d *ContrailDriver) CreateNetwork(req *network.CreateNetworkRequest) error
 		fmt.Printf("%v: %v\n", k, v)
 	}
 
-	return nil
+	reqGenericOptionsMap, exists := req.Options["com.docker.network.generic"]
+	if !exists {
+		return errors.New("Generic options missing")
```

Comment:
Not really. If I return request to docker daemon, I don't log errorsbecause docker daemon will do it for me. Else, I log errors.